src/Jupyter/Jupyter.csproj

```diff
@@ -22,11 +22,12 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Jupyter.Core" Version="1.3.52077" />
+    <PackageReference Include="Microsoft.Jupyter.Core" Version="1.3.9214-CI-20200415-173910" />
```

Comment:
I think this makes sense for a feature branch, but we should be sure to only use versions from nuget.org when merging to master. Let me know if I can help push new jupyter-core releases!

Reply:
Yes, totally agree. Probably not important immediately, but yes at some point I'll ask for your help with that!